File: C_Programming/Competitive/program30.c
//////  Write a program which accepts a number from user and display its table in revers order.

//////  Input : 2
//////  Output : 20 18  16  14  12  10  8   6   4   2

//////  Input : 5
//////  Output : 50 45  40  35  30  25  20  15  10  5

//////  Input : -5
//////  Output : 50 45  40  35  30  25  20  15  10  5

#include<stdio.h>

void TableRev(int iNo)
{
    int iCnt = 0;
    int iTab = 0;

    if (iNo < 0)
    {
        iNo = -iNo;
    }

    for (iCnt = 10; iCnt >= 1; iCnt--)
    {
        iTab = iCnt * iNo;

        printf("%d\t", iTab);
    }
}

int main()
{
    int iValue = 0;

    printf("Enter a number : \n");
    scanf("%d", &iValue);

    TableRev(iValue);
    
    return 0;
}

File: Java_Programming/Competitive/program30.java
////    Write a program which accepts a number from user and ON its first 4 bits.
////    Return the modified number.
////    Input : 73
////    Output : 79

import java.util.*;

class program30
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;
        int iRet = 0;

        System.out.println("Enter a number : ");
        iValue = sobj.nextInt();

        Bitwise bobj = new Bitwise();

        iRet = bobj.OnBit(iValue);

        System.out.println("Modified number is : " + iRet);

        sobj.close();
    }
}

class Bitwise
{
    int OnBit(int iNo)
    {
        int iMask = 0xF;

        return (iNo | iMask);
    }
}